samples/boards/sensortile_box/src/main.c

```diff
@@ -252,20 +252,20 @@ void main(void)
 	int cnt = 1;
 
 	led0 = device_get_binding(DT_ALIAS_LED0_GPIOS_CONTROLLER);
-	gpio_pin_configure(led0, DT_ALIAS_LED0_GPIOS_PIN, GPIO_DIR_OUT);
+	gpio_pin_configure(led0, DT_ALIAS_LED0_GPIOS_PIN, GPIO_OUTPUT);
```

Comment:
I overlooked this before. The sample is board specific but for consistency we could pull in the configuration flags and also use `GPIO_OUTPUT_ACTIVE`. As in `GPIO_OUTPUT_ACTIVE | DT_ALIAS_LED0_GPIOS_FLAGS`. LED1 would need to be configured with `GPIO_OUTPUT_INACTIVE`.

Reply:
> I overlooked this before. The sample is board specific but for consistency we could pull in the configuration flags and also use `GPIO_OUTPUT_ACTIVE`. As in `GPIO_OUTPUT_ACTIVE | DT_ALIAS_LED0_GPIOS_FLAGS`. LED1 would need to be configured with `GPIO_OUTPUT_INACTIVE`.

@mnkp 
I rebase this commit so that I have now GPIO_OUTPUT_ACTIVE definition.
I changed the two gpio_pin_configure() using the active/inactive ORring them with the flags in dts. Take a look if now is fine.

Reply:
Thanks, looks good!